addons/rose/tests/integration/components/rose/form/checkbox-test.js

```diff
@@ -30,6 +30,17 @@ module('Integration | Component | rose/form/checkbox', function (hooks) {
     assert.ok(await find('input'));
   });
 
+  test('it renders optional helper text', async function (assert) {
+    assert.expect(1);
+    await render(
+      hbs`<Rose::Form::Checkbox @label="Label" @helperText="Hello world" />`
+    );
+    assert.equal(
+      await find('.rose-form-helper-text').textContent.trim(),
```

Comment:
Is `await` needed for `find`? Looking through rose test files, we seem to have a mix of tests using `find` as both synchronous and asynchronous actions.

Ref: https://github.com/emberjs/ember-test-helpers/blob/master/API.md#find

Reply:
Not necessary.  It looks like a bunch of form component tests have that.  I removed for this one.